fmf/utils.py

```diff
@@ -38,6 +39,9 @@
 # Cache expiration in seconds
 CACHE_EXPIRATION = 1200
 
+# Lock timeout in seconds
+LOCK_TIMEOUT = 10
```

Comment:
Sometime the fetching can take longer. I would extend this a bit, let's say 30 seconds?

Reply:
This is something to discuss. We know it can be even minutes :/ (large repos and slow network)